Remove www. from URL, *.herokuapp.com does not match

EdHurtig opened the pull request:
Wildcard certs like *.herokuapp.com can only secure
a single level of subdomains, therefore the www.
is not able to be verified, however it is not needed

See #5

EdHurtig committed:
Remove www. from URL, *.herokuapp.com does not match

Wildcard certs like *.herokuapp.com can only secure
a single level of subdomains, therefore the www.
is not able to be verified, however it is not needed

EdHurtig committed:
Add spring framework

EdHurtig committed:
Create test model

EdHurtig committed:
Add Generic HTTPRequest Async Task

Feast Your Eyes on this beauty

EdHurtig committed:
Add Generic Reponse Wrapper class to wrap the model

EdHurtig committed:
Remove PostTask

EdHurtig committed:
Clean up a bit

EdHurtig commented:
B/C Spring sucks and is way more complicated than it needs to be I'm switching us over to something simple: http://stackoverflow.com/questions/3197335/restful-api-service

EdHurtig committed:
Atomic Commit of Current State

ColinKohler commented:
This looks good, provided it works. FIx the merge conflicts and we should be good to go.

EdHurtig committed:
Another Blast Commit

EdHurtig committed:
Add User Presentable Exception

EdHurtig committed:
Rename DataError to UserDataError

EdHurtig committed:
Extend RuntimeException for UserPresentableException

EdHurtig committed:
Delete old unused files from exploration

EdHurtig committed:
Move APIMeta out to it's own new class called ResponseMeta

Note that all refactoring comes in the next commit titled "Document All The Things"

EdHurtig committed:
Document All The Things

EdHurtig committed:
Add RestUtils Class to handle REST Utility things

EdHurtig committed:
Massively clean up the CreateAccountActivity

EdHurtig committed:
Merge branch 'master' into ed/ssl-fixes

* master:
  Minor size changes to popup
  Added basic popup logic
  Small changes to password change activity
  Added Change Password and created whole row buttons
  remove parts related to firstname and lastname
  remove firstname and lastname
  remove first and last name keep the username age and gender
  Remove firstname and lastname change layout of birthday below email
  Added Meditation and Lesson again
  Preferences UI Changes

Conflicts:
	app/src/main/java/justbe/mindfulnessapp/CreateAccountActivity.java
	app/src/main/res/layout/content_getting_started.xml
